app/code/Magento/Review/Model/ResourceModel/Review/Summary.php

```diff
@@ -73,4 +74,44 @@ public function reAggregate($summary)
         }
         return $this;
     }
+
+    /**
+     * Append review summary fields to product collection
+     *
+     * @param \Magento\Catalog\Model\ResourceModel\Product\Collection $productCollection
+     * @param int $storeId
+     * @param string $entityCode
+     * @return Summary
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function appendSummaryFieldsToCollection(
+        \Magento\Catalog\Model\ResourceModel\Product\Collection $productCollection,
+        $storeId,
+        $entityCode
+    ) {
+        if (!$productCollection->isLoaded()) {
+            $sumEntitySubSelect = $this->getConnection()->select();
```

Comment:
@Den4ik one occurrence left - `sumEntitySubSelect` - please fix it.

Reply:
@orlangur Thank you for attention. I'll fix it too